app/build.gradle

```diff
@@ -44,7 +43,7 @@ android {
 
     buildTypes {
         debug {
-            minifyEnabled false
+            minifyEnabled true
```

Comment:
It is needed for security but I prefer to have it in different PR

Reply:
With minifyEnabled false we have ` Total methods in app-debug.apk: 57762 (88.14% used)`

Reply:
Usually, you can not just switch on proguard, it requires checking if app still working as it was before, if we need to adjust default settings, if we want to add more specific to our project, etc

Reply:
Ok, got it.